apollo-normalized-cache/src/main/java/com/apollographql/apollo/cache/normalized/internal/RealApolloStore.java

```diff
@@ -120,11 +113,9 @@ public RealApolloStore(@NotNull NormalizedCache normalizedCache, @NotNull CacheK
   @Override @NotNull public ApolloStoreOperation<Boolean> clearAll() {
     return new ApolloStoreOperation<Boolean>(dispatcher) {
       @Override public Boolean perform() {
-        return writeTransaction(new Transaction<WriteableStore, Boolean>() {
-          @Override public Boolean execute(WriteableStore cache) {
-            optimisticCache.clearAll();
-            return Boolean.TRUE;
-          }
+        return writeTransaction(cache -> {
```

Comment:
Just applied IntelliJ's suggestion to convert these to lambdas. I can revert these if not wanted.